apps/web-giddh/src/app/daybook/export-daybook/export-daybook.component.ts

```diff
@@ -19,6 +19,7 @@ export class ExportDaybookComponent implements OnInit, OnDestroy {
     public emailTypeSelected: string = '';
     public emailTypeMini: string = '';
     public emailTypeDetail: string;
+    public emailTypeExpanded: string;
```

Comment:
add doucmentation

Reply:
done

Reply:
Write documentation for  each line of variable declaration

Reply:
done